app/assets/stylesheets/base/_forms.scss

```diff
@@ -36,3 +36,9 @@ span.error {
 #trip_start_date_3i {
   max-width: fit-content;
 }
+
+.trips_table_index {
```

Comment:
Does this belong in `_forms.scss`? or would it be more suitable in `_tables.scss`?

Reply:
Done in sha `3906085` with fixup